rust/Makefile

```diff
@@ -11,9 +11,6 @@ always-$(CONFIG_RUST) += exports_core_generated.h
 obj-$(CONFIG_RUST) += helpers.o
 CFLAGS_REMOVE_helpers.o = -Wmissing-prototypes -Wmissing-declarations
 
-always-$(CONFIG_RUST) += libmacros.so
-no-clean-files += libmacros.so
-
```

Comment:
I moved this around, because as far as I can tell, at this point it's not known if rustc is installed. Is that a problem?

Reply:
I think it should be OK.